contracts/tokens/UnwETH.sol

```diff
@@ -36,6 +35,13 @@ contract UnwEth is Authorizable, Destructable {
         deposit();
     }
 
+    // Todo Remove
+    function addAuthorizedAddress(address target)
+        public
```

Comment:
Is that normal?

Reply:
This is not good if we actually deploy this token. As we currently not plan to do that, I did not care enough to solve it properly